spec/features/user_groups_spec.rb

```diff
@@ -3,12 +3,14 @@
 feature 'User groups CRUD' do
   scenario 'Create' do
     admin = create(:admin)
-    login_as(admin)
+    user = create(:user)
 
+    login_as(admin)
     visit new_user_group_path
 
     fill_in 'Description', with: 'department name'
     fill_in 'Name', with: 'team name'
+    select user.email, from: 'Users'
```

Comment:
what do you mean? like check that the user is added to the group? I figured we'd do that when we show the users on the user group index page....which I think you are working on?